src/Cli/Commands/ConfigureOptions.cs

```diff
@@ -59,6 +67,18 @@ public ConfigureOptions(
         [Option("runtime.graphql.depth-limit", Required = false, HelpText = "Max allowed depth of the nested query. Allowed values: (0,2147483647] inclusive. Default is infinity. Use -1 to remove limit.")]
         public int? DepthLimit { get; }
 
+        [Option("runtime.graphql.enabled", Required = false, HelpText = "Enable DAB's GraphQL endpoint in Runtime Config. Allowed values: true(default) or false (boolean).")]
+        public bool? RuntimeGraphQLEnabled { get; }
+
+        [Option("runtime.graphql.path", Required = false, HelpText = "Customize DAB's GraphQL endpoint path in Runtime Config. Allowed values: string. Conditions: Starting with '/', no spaces and no reserved characters.")]
+        public string? RuntimeGraphQLPath { get; }
+
+        [Option("runtime.graphql.allow-introspection", Required = false, HelpText = "Allow/Deny GraphQL introspection requests in Runtime Config. Allowed values: true(default) or false (boolean).")]
+        public bool? RuntimeGraphQLAllowIntrospection { get; }
+
+        [Option("runtime.graphql.multiple-mutations.create.enabled", Required = false, HelpText = "Enable/Disable multiple-mutation create operations on DAB's generated GraphQL schema in Runtime Config. Allowed values: true(default) or false (boolean).")]
+        public bool? RuntimeGraphQLMultipleMutationsCreateEnabled { get; }
+
```

Comment:
```suggestion
        [Option("runtime.graphql.enabled", Required = false, HelpText = "Enable DAB's GraphQL endpoint in Runtime Config. Default: true (boolean).")]
        public bool? RuntimeGraphQLEnabled { get; }

        [Option("runtime.graphql.path", Required = false, HelpText = "Customize DAB's GraphQL endpoint path in Runtime Config. Allowed values: string. Conditions: Prefix with '/', no spaces and no reserved characters.")]
        public string? RuntimeGraphQLPath { get; }

        [Option("runtime.graphql.allow-introspection", Required = false, HelpText = "Allow/Deny GraphQL introspection requests in Runtime Config. Default: true (boolean).")]
        public bool? RuntimeGraphQLAllowIntrospection { get; }

        [Option("runtime.graphql.multiple-mutations.create.enabled", Required = false, HelpText = "Enable/Disable multiple-mutation create operations on DAB's generated GraphQL schema in Runtime Config. Default: true (boolean).")]
        public bool? RuntimeGraphQLMultipleMutationsCreateEnabled { get; }

```

If you don't prefer the above suggestion, make sure spacing is consistent between example values and the commentary in the parenthesis:
```csharp
Allowed values: true (default) or false (boolean).
```

The motivation behind my suggestion was to condense intended meaning in fewer words:
`Default: true (boolean).` implies that false the other valid value.

Reply:
this works. Modified